.github/workflows/build-x86-image.yaml

```diff
@@ -1264,3 +1264,69 @@ jobs:
           docker push kubeovn/centos7-compile-dev:$TAG-x86
           # docker push kubeovn/centos8-compile:$TAG-x86
           # docker push kubeovn/centos8-compile-dev:$TAG-x86
+
+  kube-ovn-submariner-conformance-e2e:
+    name: Kube-OVN Submariner Conformance E2E
+    needs:
+      - build-kube-ovn
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Create the default branch directory
+        if: (github.base_ref || github.ref_name) != github.event.repository.default_branch
+        run: mkdir -p test/e2e/source
+
+      - name: Check out the default branch
+        if: (github.base_ref || github.ref_name) != github.event.repository.default_branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          fetch-depth: 1
+          path: test/e2e/source
+
+      - name: Export E2E directory
+        run: |
+          if [ '${{ github.base_ref || github.ref_name }}' = '${{ github.event.repository.default_branch }}' ]; then
+            echo "E2E_DIR=." >> "$GITHUB_ENV"
+          else
+            echo "E2E_DIR=test/e2e/source" >> "$GITHUB_ENV"
+          fi
+
+      - name: Install kind
+        uses: helm/kind-action@v1
+        with:
+          install_only: true
+
+      - name: Install submariner subctl
+        run: |
+          curl -Ls https://get.submariner.io | VERSION=v0.13.4 bash
+          sudo chmod +x ~/.local/bin/subctl
+          sudo mv ~/.local/bin/subctl /usr/bin/
```

Comment:
We should move subctl installation to Makefile so that we can run the test on PC easily.

Reply:
on my local env , I need to change http-proxy for the curl command, and change http-proxy to "",  when it use subctl command join cluster.